client/components/social-icons/google.jsx

```diff
@@ -19,13 +20,14 @@ export default class GoogleIcon extends PureComponent {
 	};
 
 	render() {
-		const props = omit( this.props, [ 'isDisabled' ] );
+		const props = omit( this.props, [ 'isDisabled', 'isLoading' ] );
```

Comment:
While we're touching this code, it would be nice to remove the usage of `_.omit` in favor of something like:
```js
const { isDisabled, isLoading, ...props } = this.props;
```